src/fairchem/core/datasets/lmdb_dataset.py

```diff
@@ -50,20 +52,21 @@ class LmdbDataset(Dataset[T_co]):
     """
 
     def __init__(self, config) -> None:
-        super().__init__()
-        self.config = config
+        super().__init__(config)
 
         assert not self.config.get(
             "train_on_oc20_total_energies", False
         ), "For training on total energies set dataset=oc22_lmdb"
 
-        self.path = Path(self.config["src"])
+        assert (
+            len(self.paths) == 1
+        ), f"{type(self)} does not support a list of src paths."
+        self.path = self.paths[0]
```

Comment:
Why is a list being fed in here in the first place?

Reply:
This is a good point. I don't know, i assumed it was because some datasets might take multiple files, potentially multiple lmdb paths. But i dont think this is currently used, I will take this out.

Reply:
On second pass, looks like we do use multiple paths for ASE datasets, 
https://github.com/FAIR-Chem/fairchem/blob/main/tests/core/datasets/test_ase_datasets.py#L89